### ISSUE TITLE
[NEWS] Automatyczne wznawianie ofert

### ISSUE THREAD

**Issue by PawelTaberski** [allegro/allegro-api#1869]
Od dziś możesz automatycznie wznowić ofertę i licytację:

- ofertę wznowimy ze stałą początkową liczbą przedmiotów niezależnie od tego, ile przedmiotów sprzedasz
- licytację wznowimy tylko, gdy nie zakończyła się sprzedażą.

Jeśli chcesz, aby oferta lub licytacja wznowiła się automatycznie, to prześlij w jej strukturze:

```
    "publication": {
    	"republish": true
    }
```

Więcej na temat tego jak wystawić ofertę znajdziesz w [naszym poradniku](https://developer.allegro.pl/sale/#step-8-complete).

**Comment by kfijalko** [comment]
1) W  jakich przypadkach oferta się wznawia automatycznie? Jakie powody zakończenia?
2) Czy zachowa zestawy, warianty, promowania?

**Comment by kormichu** [comment]
Czy to będzie działać na takiej samej zasadzie jak w SOAP API że tworzona jest aukcja z nowym numerkiem czy nie ?

**Comment by PawelTaberski** [comment]
Tak, dokładnie jak napisał @kormichu będzie to działać tak jak dotychczas. I jak oferta miała ustawione autowznawianie przez WebAPI to zwrócimy tam tą również tą informację.

**Comment by Onixarts** [comment]
No i brawo, ale...

Czy ta stała, początkowa liczba sztuk jest gdzieś zwracana? I dlaczego akurat tak, bo wydaje się, że większy sens miało by ponowne wystawienie z niesprzedaną ilością. Tak ktoś wystawi 10 sztuk, sprzeda mu się 9 i ponownie wystawi mu się 10 sztuk - chyba, że źle to zrozumiałem, i będzie dym do nas, że nasz soft wystawił towar, którego nie ma :)

**Comment by PawelTaberski** [comment]
Jest to początkowa liczba przedmiotów z jaką wystawiłeś daną ofertę. 
Dokładnie tak samo jak działało to dotychczas, nic nie zmieniło się w sposobie działania tej opcji i działa to identycznie jak na froncie.
Obecnie nigdzie nie zwracamy informacji o początkowej liczbie przedmiotów z jaką wystawiłeś daną ofertę.

**Comment by lukasnet** [comment]
Czy ta informacja o wznawianiu "republish" będzie zwracana również w sale/offers? bo jak na razie jej tam nie ma

**Comment by PawelTaberski** [comment]
Ta informacja jest zwracana tylko przy pobraniu danej oferty. Powiedz mi do czego potrzebujesz tej informacji w zasobie sale/offers.

**Comment by lukasnet** [comment]
Do sprawdzania czy oferty mają tą opcję ustawioną, nie trzeba by pobierać wszystkich danych aukcji, a to szybsze działanie i mniejsza ilość zasobów.

**Comment by PawelTaberski** [comment]
Rozumiem przekazałem taką sugestię do zespołu, który wdrażał tą funkcję.
